server/app/controllers/dev/DatabaseSeedController.java

```diff
@@ -267,6 +417,17 @@ private ProgramDefinition insertProgramWithBlocks(String name, QuestionDefinitio
               PredicateExpressionNode.create(operation), PredicateAction.SHOW_BLOCK);
       programDefinition = programService.setBlockPredicate(programId, blockId, predicate);
 
+      // Add file upload as optional since it does not work for local testing.
```

Comment:
Not blocking: This *should* work for local testing. I'm happy to help debug if things aren't working though.

Reply:
Updated this comment. We'll resolve this offline!